avocado/core/xunit.py

```diff
@@ -199,9 +199,7 @@ def end_test(self, state):
             self.xml.add_skip(state)
         elif status == 'FAIL':
             self.xml.add_failure(state)
-        elif status == 'ERROR':
-            self.xml.add_error(state)
-        elif status == 'INTERRUPTED':
+        else:   # ERROR, INTERRUPTED, ...
```

Comment:
I missed that. I'm not sure I agree. I think it's worth keeping this manual as we might want to double-check the status is mapped to pass in the result plugin. @clebergnu what is your opinion?